README.md

```diff
@@ -94,3 +94,7 @@ Please report security issues to the module maintainers directly. Please don't f
 
 If you would like to make contributions to the module please ensure you raise a pull request and discuss with the
 module maintainers.
+
+## Notes
+
+SilverStripe 3 is approaching end of life (EOL) in terms of support. This backport has been made specifically to help protect older systems while they undergo upgrade projects, as time is often a factor with budgeting. In this regard support will be limited for this version of the TOTP Authenticator, however if your issue is still applicable in the latest 4.x release please do log the issue and/or create a pull request to resolve the issue against the relevant newer branch.
```

Comment:
I think that this is a really good addition, but we should just refer back to our support obligations for SS3 modules in general. We don't really need to justify why we've backported.

```suggestion
Unfortunately, [SilverStripe 3 has entered limited support in June 2018](https://www.silverstripe.org/blog/update-on-silverstripe-5-x/). This means we'll only be fixing critical bugs and security issues for SilverStripe 3 going forward.

You can read the [SilverStripe Roadmap](https://www.silverstripe.org/software/roadmap/) for more information on our support commitments.
```